setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name =  flask-container-scaffold
+version = 0.0.1
+author = Jason Guiditta
+author_email = jason.guiditta@gmail.com
+description = Configuration layer to aid in deployment of Flask apps
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/release-depot/flask-container-scaffold
+project_urls =
+    Bug Tracker = https://github.com/release-depot/flask-container-scaffold/issues
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
```

Comment:
This looks a little odd and that might be because of it being an ini. The 2 equal signs seems odd.

Reply:
Seems to come straight [from the doc in the README](https://packaging.python.org/tutorials/packaging-projects/#configuring-metadata), as peculiar as it looks...

Reply:
Thanks for finding that. It's very odd looking, but if that is how it work then so be it.